fda-model/compute/RandomProvider.cs

```diff
@@ -6,7 +6,7 @@ namespace HEC.FDA.Model.compute
     public class RandomProvider : IProvideRandomNumbers
     {
         private Random _randomNumberGenerator;
-        private int _seed;
+        private int _seed = 1234;
```

Comment:
We use seeds all over. Should we add the generic seed to the Integers Constants class and just make it static?